app/controllers/trips_controller.rb

```diff
@@ -1,6 +1,7 @@
 class TripsController < ApplicationController
   def index
-    @trips = Trip.all
+    @current_trips = Trip.where("end_date > ?", Time.now).order("start_date")
+    @past_trips = Trip.where("end_date <= ?", Time.now)
```

Comment:
A few suggestions:
1. move the query to the model (use named scope), so that you can call `Trip.current` and `Trip.past`
2. make the order a scope as well, i.e. `Trip.by_start_date`
3. consider creating a `TripsPresenter`, which has current and past trips.

Reply:
Once you have the scopes on the model, you can stub them in a spec. For example:
```
past_trip = build_stubbed(:trip, start_date: 1.month.ago)
allow(Trip).to receive(:past).and_return([past_trip])
```

Reply:
Lastly, never use `Time.now` which uses your machine time. Instead either use `Time.current` or `Time.zone.now`, which will use the application time zone

Reply:
Actually, thinking about this more, I would leave creating the presenter for now. But will still do points 1 and 2

Reply:
- Added scope in the model in sha:`a0599c2` & add the scope into the index#TripsConroller in  she: `2bd2df1` and replace `Time.now` with `Time.current`. Not sure the convention for the position of scope, I put it just before the private methods;
- Added the a sub_header with tab for `current trips` , `past trips` and `all trips` has been made in few commits (last 4), I will add a pic in the PR description.

But I still need to implement the test :)